examples/demo-app/src/cloud-providers/fsq-studio/fsq-studio-provider.js

```diff
@@ -0,0 +1,177 @@
+import FSQIcon from './fsq-studio-icon';
+import {Provider} from 'kepler.gl/cloud-providers';
+import {Auth0Client} from '@auth0/auth0-spa-js';
+import {KEPLER_FORMAT} from '@kepler.gl/cloud-providers';
+
+const NAME = 'FSQ-Studio';
+const DISPLAY_NAME = 'Foursquare Studio';
+const APP_NAME = 'Kepler.gl';
+
+const STUDIO_PRIVATE_STORAGE_ENABLED = true;
+const STUDIO_AUTH_AUDIENCE = 'https://foursquare.com/api/';
+const STUDIO_AUTH_SCOPE = 'openid profile email';
+
+const STUDIO_KEPLER_GL_IMPORT_SOURCE = 'kepler.gl-raw';
```

Comment:
What does kepler.gl-raw mean? Is it a URL path, a parameter value, etc? Maybe add comment before this constant,